hack/init_data_plane.sh

```diff
@@ -20,32 +20,27 @@
 # The current project set in gcloud MUST be the same as where the cluster is running.
 # The script always uses the same service account called cre-pubsub.
 
-SERVICE_ACCOUNT=cre-pubsub
-KEY_TEMP=cre-pubsub.json
-PROJECT_ID=$(gcloud config get-value project)
-NAMESPACE=default
-if [[ -z "$1" ]]; then
-  echo "NAMESPACE not provided, using default"
-else
-  NAMESPACE="$1"
-  echo "NAMESPACE provided, using ${NAMESPACE}"
-  kubectl create namespace $NAMESPACE
-fi
+source $(dirname $0)/lib.sh
 
-# Create the service account for the data plane
-gcloud iam service-accounts create ${SERVICE_ACCOUNT}
+readonly NAMESPACE="default"
```

Comment:
Is `NAMESPACE` really readonly? Line 31 sets it again.

Reply:
This is a bug. I will fix it. This variable should not be readonly.